development/docker-compose.bot-framework.yml

```diff
@@ -0,0 +1,8 @@
+---
+services:
+  nautobot:
+    extra_hosts:
+      - "localhost:host-gateway" # Allow Nautobot to communicate with the Bot Framwork Emulator running on the host machine.
+  worker:
+    extra_hosts:
+      - "localhost:host-gateway" # Allow Nautobot to communicate with the Bot Framwork Emulator running on the host machine.
```

Comment:
Is this required because there's some hard-coded `host-gateway` url that we need to override?

Reply:
This part was from Jeff, so this is just my basic understanding. Bot Framework Emulator wants to be setup using ngrok, but an alternative (that Jeff put docs for) is to use localhost, so we would point Nautobot to that.

Reply:
@jeffkala can you add some more details here?